boards/arm/beagle_play_cc1352/doc/index.rst

```diff
@@ -0,0 +1,161 @@
+| DIO10 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO11 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO12 | CC1352_RX    | AM62 UART6_TXD                      |
++-------+--------------+-------------------------------------+
+| DIO13 | CC1352_TX    | AM62 UART6_RXD                      |
++-------+--------------+-------------------------------------+
+| DIO14 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO15 | CC1352_BOOT  | AM62 GPIO0_13                       |
++-------+--------------+-------------------------------------+
+| DIO16 | CC1352_TDO   | TAG-CONNECT TDO                     |
++-------+--------------+-------------------------------------+
+| DIO17 | CC1352_TDI   | TAG-CONNECT TDI                     |
++-------+--------------+-------------------------------------+
+| DIO18 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO19 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO20 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO21 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO22 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO23 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO24 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO25 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO26 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO27 | LED1         | CC1352_LED1 yellow LED9             |
++-------+--------------+-------------------------------------+
+| DIO28 | LED2         | CC1352_LED2 yellow LED8             |
++-------+--------------+-------------------------------------+
+| DIO29 | RF_PA        | Antenna mux PA enable               |
++-------+--------------+-------------------------------------+
+| DIO30 | RF_SUB1G     | Antenna mux SubG enable             |
++-------+--------------+-------------------------------------+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+To flash, disable the existing driver that ties up the serial port and use
+the customized BSL Python script.
+
+* https://docs.beagleboard.org/latest/boards/beagleplay/demos-and-tutorials/zephyr-cc1352-development.html
+
```

Comment:
Can't you make `west flash` work for this target upstream? I think just using the disable bcfserial overlay and having the flashing script should fix it. Pointing to the above documentation is not really useful for someone trying to develop upstream. Those instructions are specific to downstream repo/there is no specific instructions for independently flashing.

```
Make sure you are in Zephyr directory
cd $HOME/bcf-zephyr (why should someone have a copy of downstream SDK to develop/flash upstream?)
Flash Blinky
cc2538-bsl.py build/blinky
```

@Ayush1325 , please fix this

Reply:
So, the bcfserial overlay would need to be loaded through uboot which means a restart is required. This also would need flasher script whose exclusion was a conscious decision.
I think there are ways to apply overlays dynamically, but haven't tried them. 
I am working on a greybus based solution for flashing and updating firmware.

Reply:
Restart required to flash the device should have nothing to do with having a reliable way to flash the device. A board submitted upstream should have reliable way to build and flash otherwise it is not much useful to other developers who start development.

Reply:
Glad to hear about the effort on greybus based FW update mechanism, that would be nice, but won't those be for the remote nodes/BCF. I remember porting MCUBoot for the CC1352 and using mcumgr for updates (but again reboot will be a concern)